Dockerfile

```diff
@@ -1,35 +1,56 @@
-# build stage
-FROM golang:1-alpine AS build-env
-RUN apk add --no-cache \
-    build-base \
-    linux-headers \
-    coreutils \
-    binutils \
-    libtool \
-    musl-dev \
-    git \
-    make \
-    gcc \
-    libc-dev \
-    zip \
-    ca-certificates \
-    libcap
-
-ENV GO111MODULE=on \
-    CGO_ENABLED=0
-    
-WORKDIR /src
+# get newest certificates
+FROM --platform=$BUILDPLATFORM alpine:3.16 AS ca-certs
+RUN apk add --no-cache ca-certificates
+RUN --mount=type=cache,target=/etc/ssl/certs \
+    update-ca-certificates 2>/dev/null || true
```

Comment:
Can you please add a comment for why we need ` || true`?

Reply:
Added clarifying comment